Assets/MRTK/SDK/Experimental/ScrollingObjectCollection/ScrollingObjectCollection.cs

```diff
@@ -431,51 +442,59 @@ private float maxY
         // Maximum amount the scroller can travel (horizontally) - this will always be zero. Here for readability
         private readonly float maxX = 0.0f;
 
-        // Depth of scrolling background box collider
-        private float scrollingBackgroundDepth = 0.001f;
-
         // Minimum amount the scroller can travel (horizontally)
-        private float minX
+        private float MinX
         {
             get
             {
-                return NodeList.Count != 0 ? Mathf.Ceil(SafeDivisionFloat(NodeList.Count - (ViewableArea * ItemsPerTier), ItemsPerTier)) * CellWidth * -1.0f : 0.0f;
+                return contentMaxBounds == null || contentMaxBounds.size.x <= 0 ? 0 :
+                     Mathf.Max(0, SafeDivisionFloat(contentMaxBounds.size.x, transform.lossyScale.x) - TiersPerPage * CellWidth) * -1.0f;
             }
         }
 
+        // Depth of scrolling background box collider
+        private float scrollingBackgroundDepth = 0.001f;
```

Comment:
>private float scrollingBackgroundDepth = 0.001f; [](start = 8, length = 48)

is this a number users might want to modify?

Reply:
I have updated the description, as it is used kind of like a plane, so no need to configure this.